avro-fastserde/src/test/java/com/linkedin/avro/fastserde/FastSpecificDeserializerGeneratorTest.java

```diff
@@ -30,6 +31,8 @@
 
 import static com.linkedin.avro.fastserde.FastSerdeTestsSupport.*;
 
+import sun.net.www.content.text.Generic;
```

Comment:
This import was added accidentally, right?

Reply:
Yes, sorry, I must have overlooked that. I'll fix that promptly.